docs/wallet/Desktop-Wallet-Guide.md

```diff
@@ -40,3 +40,68 @@ Due to the nature of UTXO, every time a transaction is made a couple of new Unsp
 
 [latest-release]: https://github.com/alephium/desktop-wallet/releases/latest
+
+## Passphrase
+
+:::info Advanced security feature
+Please, read the following documentation before deciding to use this feature.
+:::
+
+:::tip Important takeaways
+
+- The Passphrase is an advanced security feature that adds an **additional word** that you choose to your existing secret phrase.
+- Using a Passphrase will cause an **entirely new wallet** to be created which cannot be accessed via the secret phrase alone.
+- Apart from adding another layer of security, the Passphrase grants you **plausible deniability** when under duress.
+- If you decide to use a Passphrase, it is vital to store it securely and to **remember it perfectly**, character by character.
+
+:::
+
+Assuming you have created a wallet using the desktop wallet app, you have a list of 24 words referred to as your _secret phrase_. This phrase can be used to restore your wallet and access your funds. If this 24-word secret phrase gets stolen, the attacker will be able to steal your funds. To enhance the security of our desktop wallet users and to prevent loss of funds due to theft of the 24-word secret phrase, we implemented the [BIP39 passphrase][bip39-spec] feature.
+
+The Passphrase is an additional optional 25th word that you are free to choose for yourself. It can be composed of any lower/upper-case character, number, and/or signs and be as long as you want.
+
+Once you create a wallet into the desktop wallet app, the next time you open the app to log in to your wallet, you will be able to provide an additional Passphrase. **It is important to remember that any unique Passphrase will generate and give access to a completely new wallet.**
+
+### Example scenario
+
+#### The "normal wallet"
+
+Let's assume that you created a new wallet, you gave it the name _My Alephium Wallet_ and the password _myVeryStrongPassw0rd_. Let's also assume that the generated 24-word secret phrase is `scan pause slender around cube flavor neck shrug gadget ramp rude lend capable tone nose unhappy gift across cluster minor tragic fever detail script`. You decide to transfer 100 ALPH to it. We'll call this the "normal wallet".
+
+#### The "hidden wallet"
+
+The next time you open your wallet, you select that you want to login to _My Alephium Wallet_ and you provide your password _myVeryStrongPassw0rd_. Instead of clicking _Login_ directly, however, you enter the Passphrase _letsUnlockAHiddenWallet_, and then click _Login_. This gives you access to a completely new wallet (let's call it "hidden wallet"). You will not be able to see your balance of 100 ALPH there, since the wallet you tranferred your 100 ALPH to was the one that did not use the Passphrase _letsUnlockAHiddenWallet_ (a.k.a. the "normal wallet"). You decide to transfer 10'000 ALPH to the "hidden wallet".
```

Comment:
you don't see these @nop33 ?

Reply:
> This doesn't describe the process of clicking "Optional passphrase"

The UI is not finalized yet and I don't think we have to be so explicit. WDYT?

Reply:
I think we could reformulate somewhat, because as it stand now, if i RTFM before getting into it, i will not see what it gets me to expect in the desktop wallet.

Reply:
I think we have to be explicit too... It immediately stood out to me - the doc can always change to reflect the UI and given the importance of understanding the feature I believe it's necessary

Reply:
I've updated the wording